client/app/components/dynamic-parameters/DynamicParameters.less

```diff
@@ -0,0 +1,18 @@
+.dynamic-value {
+  & ::placeholder, & .dynamic-icon {
+    color: #1890ff !important
+  }
+
+  .ant-calendar-range-picker-separator,
+  .ant-calendar-range-picker-input:not(:first-child) {
+    display: none;
+  }
```

Comment:
To prevent text clipping
<img width="214" alt="Screen Shot 2019-07-01 at 8 31 47" src="https://user-images.githubusercontent.com/486954/60412690-ca41da80-9bda-11e9-94f5-fde012280985.png">


add this:
```less
.dynamic-value {
  .ant-calendar-range-picker-input {
    width: auto;
    text-align: left;
  }
}
```

Reply:
`width: 100%` keeps some flexibility while fixing this too :)